CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.17.1-beta] - 2019-07-04
```

Comment:
You will need to remove this before releasing it

Reply:
Yeeeees sir